frontend/src/config/types.ts

```diff
@@ -372,6 +372,7 @@ interface FeatureInfoProps {
   type: DataType;
   dataTitle: string;
   labelMap?: { [key: string]: string };
+  metadata?: string;
```

Comment:
could this be types with the keys of `PopupMetaData`? This will make it easier to understand what to change in the future